pkgs/applications/networking/browsers/brave/default.nix

```diff
@@ -41,63 +40,61 @@
 , xorg
 , zlib
 , xdg-utils
-, wrapGAppsHook
+, snappy
+, libopus
+
+# command line arguments which are always set e.g "--disable-gpu"
 , commandLineArgs ? ""
+
+# Necessary for USB audio devices.
+, pulseSupport ? true, libpulseaudio ? null
+
+# For video acceleration via VA-API (--enable-features=VaapiVideoDecoder,VaapiVideoEncoder)
+, libvaSupport ? true, libva
+, enableVideoAcceleration ? libvaSupport
+
+# For Vulkan support (--enable-features=Vulkan)
+, vulkanSupport ? false, addOpenGLRunpath
+, enableVulkan ? vulkanSupport
 }:
 
-let
+with lib;
```

Comment:
Please revert this. Doing with X over an entire file is an anti pattern.

Reply:
Would `inherit (lib) whatever is needed;` be an okay solution?

Reply:
I don't like it but it is better than with over the entire file. My favorite is to always write out lib except for local with's where it is used many times.

Reply:
I've changed it to the `inherit` option, hopefully it's acceptable.